packages/jest-fake-timers/src/modernFakeTimers.ts

```diff
@@ -93,20 +92,48 @@ export default class FakeTimers {
     }
   }
 
-  useFakeTimers(): void {
-    if (!this._fakingTime) {
-      const toFake = Object.keys(this._fakeTimers.timers) as Array<
-        keyof FakeTimerWithContext['timers']
-      >;
-
-      this._clock = this._fakeTimers.install({
-        loopLimit: this._maxLoops,
-        now: Date.now(),
-        toFake,
+  useFakeTimers(fakeTimersConfig?: Config.FakeTimersConfig): void {
+    if (this._fakingTime) {
+      this._clock.uninstall();
+    }
+
+    const toSinonFakeTimersConfig = (
+      fakeTimersConfig?: Config.FakeTimersConfig,
+    ): SinonFakeTimersConfig => {
+      fakeTimersConfig = {
+        ...this._config.fakeTimers,
+        ...fakeTimersConfig,
+      } as Config.ModernFakeTimersConfig;
+
+      const advanceTimeDelta =
+        typeof fakeTimersConfig.advanceTimers === 'number'
+          ? fakeTimersConfig.advanceTimers
+          : undefined;
+
+      const toFake = new Set(
+        Object.keys(this._fakeTimers.timers) as Array<FakeableAPI>,
+      );
+
+      fakeTimersConfig?.doNotFake?.forEach(nameOfFakeableAPI => {
+        // @ts-expect-error: TODO remove after merging https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59269
+        toFake.delete(nameOfFakeableAPI);
       });
 
-      this._fakingTime = true;
-    }
+      return {
+        advanceTimeDelta,
+        loopLimit: fakeTimersConfig.timerLimit,
+        now: fakeTimersConfig.now ?? Date.now(),
+        shouldAdvanceTime: Boolean(fakeTimersConfig.advanceTimers),
+        shouldClearNativeTimers: true,
```

Comment:
Should this be configurable? Looking at the example in the issue (https://github.com/sinonjs/fake-timers/issues/406) which was the reason for this option, it felt like hardcoding `true` is perfectly fine.